File: rss-reader/app/src/main/java/com/abubusoft/kripton/examples/rssreader/service/api/VogellaApi.java
package com.abubusoft.kripton.examples.rssreader.service.api;

import com.abubusoft.kripton.examples.rssreader.service.model.RssFeed;

import retrofit2.Call;
import retrofit2.http.GET;

public interface VogellaApi {

    @GET("article.rss")
    Call<RssFeed> loadRSSFeed();
}

File: kotlin-rss-reader/app/src/main/java/com/abubusoft/rssreader/service/api/VogellaApi.kt
package com.abubusoft.rssreader.service.api

import com.abubusoft.rssreader.service.model.RssFeed
import retrofit2.Call
import retrofit2.http.GET

interface VogellaApi {

    @GET("article.rss")
    fun loadRSSFeed(): Call<RssFeed>
}